ARM/ARM-Template-Deployment.yml

```diff
@@ -29,14 +25,19 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
         
-    # Deployment of template
+    # Deployment of template    
     - name: Deploy ARM Template
       uses: azure/CLI@v1
       with:
         azcliversion: latest
         inlineScript: |
-          az group create --location ${{ env.AZURE_LOCATION }} --name ${{ env.AZURE_RESOURCE_GROUP }}
+        # You can change these environment variables for your configuration:   AZURE_RESOURCE_GROUP, AZURE_LOCATION 
+        # For details on usage of environment variables, please refer https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables
+        # or specify these parameters inline.  Example:  az group create -l westus -n MyResourceGroup
+          az group create --location ${{ env.AZURE_LOCATION }} --name ${{ env.AZURE_RESOURCE_GROUP }}   
```

Comment:
changes look fine... I would make the default deployment command include a parameter file, this is the common case.

Also, since this action only support resourceGroup deployment, I would name it accordingly

Reply:
done

Reply:
since we assert the name of the tempalte, should we also assert the name of the param file?  std practice with azuredeploy.json is to have azuredeploy.parameters.json in the same folder.  as written it would require modification 100% of the time... with a default that will work for some greater than zero set of users...

Reply:
Awesome. Makes sense.